LoopKit/Loop
# Remote overrides

Configures Loop to accept remote overrides via push notification. 

Requires a Nightscout instance running code from this branch: https://github.com/nightscout/cgm-remote-monitor/pull/5043

Testing setup:
* Clone the `remote-overrides` branch of LoopWorkspace. `git clone --branch=remote-overrides --recursive https://github.com/LoopKit/LoopWorkspace`
* Update your nightscout site to use this branch: https://github.com/nightscout/cgm-remote-monitor/pull/5043
* Log into the [Apple Developer site](https://developer.apple.com/account/resources/authkeys/list), navigate to "Certificates, IDs, and Profiles" -> "Keys" -> and click on the "+" to generate an APNs key.
* Make note of the "Key ID". 
* Download the key. It will be saved as a .p8 file, which you can click on, which will open it in TextEdit on your mac. Select all the text and copy it.
* Log in to Heroku, edit your config vars to add a new "LOOP_APNS_KEY" var and paste in the contents of your key.
* Add another new var named "LOOP_APNS_KEY_ID", and set it to your Key ID from above (it's also the name of the p8 file.
* Finally, add a third var named "LOOP_DEVELOPER_TEAM_ID", and set it to your apple developer team id. You can see this when you select your developer account to build loop, and your bundle identifier changes to something like "com.UY878SU32Q.loopkit.Loop". Your team id is the "UY878SU32Q" part.

Note: You need to have "loop" and "careportal" in your `ENABLE` for Nightscout to enable this functionality. You do *not* need to have 'openaps' in your `ENABLE` var.
